sphinxcontrib/test_reports/directives/test_case.py

```diff
@@ -74,6 +86,15 @@ def run(self):
                 case = case_obj
                 break
 
+            elif nested and case_count > -1:
```

Comment:
I like `case_count >= 0` more, as it directly shows you the first allowed value.
For me easier to read

Reply:
agreed